datumaro/cli/commands/diff.py

```diff
@@ -136,7 +136,7 @@ def _parse_comparison_method(s):
         type=float,
         help="IoU match threshold for shapes (default: %(default)s)",
     )
-    parser.add_argument(
+    distance_parser.add_argument(
```

Comment:
```suggestion
    parser.add_argument(
```
I believe this should belong to `parser`.

Reply:
No, this belongs to `distance` metric only. Please refer to [doc](https://github.com/openvinotoolkit/datumaro/blob/releases/1.2.0/docs/source/docs/command-reference/context_free/diff.md) and [code](https://github.com/openvinotoolkit/datumaro/blob/45d3f805eb2d44d433d03f6b1eb73249152a7975/datumaro/cli/commands/diff.py#L259).

Reply:
Anyway, reverted all python files changes.